### DISCUSSION TITLE
Japanese version

### DISCUSSION THREAD

**Discussion by babayoshihiko** [geocompx/geocompx.org#24] [General]
Hi there,

Can the Japanese version of Geocompr be hosted at r.geocompx.org as in Spanish and French?

**Comment by Nowosad** [comment]
Yes -- of course, @babayoshihiko. What is your current workflow? I am asking, because for the other translations: we have github actions that builds the book into the gh-pages branch, and then the content of this branch is used for the online publication. Would you be able to have something similar (at least publishing the rendered version of the book at the gh-pages branch)?

**Reply by babayoshihiko** [reply]
For the time being, I will build locally and commit HTML files to the branch "gh-pages". Is it OK?

**Reply by Robinlovelace** [reply]
That's OK for now but we should get you set-up with GitHub actions.

**Reply by babayoshihiko** [reply]
I thought that copying .github/workflows may build the HTML files. But it failed. I have levt main.yaml in workflows. Can anyone give me advice what to do?

I do not have a permission?

![image](https://github.com/user-attachments/assets/da9d7062-928a-44e1-befc-2d0ae094736a)

**Reply by Nowosad** [reply]
I will look at this next week @babayoshihiko

**Reply by Nowosad** [reply]
@babayoshihiko -- it should be working now -- https://r.geocompx.org/jp (but please double check)

**Reply by Robinlovelace** [reply]
🎉
